### PR TITLE
[new][dev] Add the new rule to the index, extends #139

### PR DESCRIPTION
**Pull request by jessebeach**
Uses JSCodeShift to add the new rule to the index file as well.

### CODE REVIEW

File: scripts/create-rule.js [hunk 1]
```diff
@@ -29,3 +30,24 @@ const docBoilerplate = docBoilerplateGenerator(ruleName);
 fs.writeFileSync(rulePath, ruleBoilerplate);
 fs.writeFileSync(testPath, testBoilerplate);
 fs.writeFileSync(docsPath, docBoilerplate);
+
+// Add the rule to the index
+exec([
+  './node_modules/jscodeshift/bin/jscodeshift.sh',
```
Comment (ljharb) [inline scripts/create-rule.js]: could this use `require.resolve('jscodeshift/bin/jscodeshift.sh')`, or better, `path.join(require.resolve('jscodeshift'), require('jscodeshift/package.json').bin.jscodeshift)`? `node_modules` should ideally never be hardcoded anywhere.

File: src/index.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = {
     'img-has-alt': require('./rules/img-has-alt'),
     'img-redundant-alt': require('./rules/img-redundant-alt'),
     'label-has-for': require('./rules/label-has-for'),
-    lang: require('./rules/lang'),
+    'lang': require('./rules/lang'),
```
Comment (ljharb) [inline src/index.js]: these are valid identifiers, why would they need to be quoted?

the airbnb linter rules require `consistent-as-needed` here, not `consistent`.

Comment (jessebeach) [inline src/index.js]: They don't need to be, but the string comparisons were being weird and it's late and I was being lazy :)

So, looking into it more, `'scope'` is a Literal and `scope` is an Identifier and of course the value accessor prop is different for the type.

Comment (ljharb) [inline src/index.js]: Gotcha - it seems prudent for any codemod in the repo to be robust against any potential object key used :-) can we make it support both?

File: README.md [hunk 3]
```diff
@@ -111,6 +111,15 @@ You can also enable all the recommended rules at once. Add `plugin:jsx-a11y/reco
 - [scope](docs/rules/scope.md): Enforce `scope` prop is only used on `<th>` elements.
 - [tabindex-no-positive](docs/rules/tabindex-no-positive.md): Enforce `tabIndex` value is not greater than zero.
 
+## Creating a new rule
```
Comment (evcohen) [inline README.md]: 👍 looks good - just to clarify, i meant automating adding to the list of rules in the README in my previous comment, but should be done in a separate PR.

Comment (jessebeach) [inline README.md]: Derp, ya, I totally misunderstood that. There's a project called remark (http://remark.js.org/) for parsing markdown, but jscodeshift is wedded to recast. It'll take a little time to abstract out the re-casters from jscodeshift, making this kind of codemodding of markdown possible.
